lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,630 @@
+    name: foo
+    rename_to: bar
+
+- name: Change the schema of an existing sequence to foobar
+  postgresql_sequence:
+    name: foobar
+    newschema: foobar
+
+- name: Change the owner of an existing sequence to foobar
+  postgresql_sequence:
+    name: foobar
+    owner: foobar
+
+- name: Drop a sequence called foobar
+  postgresql_sequence:
+    name: foobar
+    state: absent
+
+- name: Drop a sequence called foobar with cascade
+  postgresql_sequence:
+    name: foobar
+    cascade: yes
+    state: absent
+'''
+
+RETURN = r'''
+state:
+  description: Sequence state at the end of execution.
+  returned: always
+  type: str
+  sample: 'present'
+sequence:
+  description: Sequence name.
+  returned: always
+  type: str
+  sample: 'foobar'
+queries:
+    description: List of queries that was tried to be executed.
+    returned: always
+    type: str
+    sample: [ "CREATE SEQUENCE \"foo\"" ]
+schema:
+    description: Name of the schema of the sequence
+    returned: always
+    type: str
+    sample: 'foo'
+data_type:
+    description: Shows the current data type of the sequence.
+    returned: always
+    type: str
+    sample: 'bigint'
+increment:
+    description: The value of increment of the sequence. A positive value will
+                 make an ascending sequence, a negative one a descending
+                 sequence.
+    returned: always
+    type: int
+    sample: '-1'
+minvalue:
+    description: The value of minvalue of the sequence.
+    returned: always
+    type: int
+    sample: '1'
+maxvalue:
+    description: The value of maxvalue of the sequence.
+    returned: always
+    type: int
+    sample: '9223372036854775807'
+start:
+    description: The value of start of the sequence.
+    returned: always
+    type: int
+    sample: '12'
+cycle:
+    description: Shows if the sequence cycle or not.
+    returned: always
+    type: str
+    sample: 'NO'
+owner:
+    description: Shows the current owner of the sequence.
+    returned: always
+    type: str
+    sample: 'postgres'
+newname:
+    description: Shows the new sequence name after rename.
+    returned: on success
+    type: str
+    sample: 'barfoo'
+newschema:
+    description: Shows the new schema of the sequence after schema change.
+    returned: on success
+    type: str
+    sample: 'foobar'
+'''
+
+
+try:
+    from psycopg2 import __version__ as PSYCOPG2_VERSION
+    from psycopg2.extensions import ISOLATION_LEVEL_READ_COMMITTED as READ_COMMITTED
+    from psycopg2.extras import DictCursor
+except ImportError:
+    # psycopg2 is checked by connect_to_db()
+    # from ansible.module_utils.postgres
+    pass
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.database import SQLParseError, pg_quote_identifier
+from ansible.module_utils.postgres import connect_to_db, postgres_common_argument_spec
+from ansible.module_utils._text import to_native
+from ansible.module_utils.six import iteritems
+
+
+class Sequence(object):
```

Comment:
```suggestion
class Sequence(object):

    """Maybe docstrings for classes and methods using PEP257
    
    ...
    """

```
It's just a proposal. I can do it by myself later but you would do it faster as an author than me.